core/src/test/scala/akka/persistence/cassandra/query/EventsByTagSpec.scala

```diff
@@ -851,3 +851,42 @@ class EventsByTagStrictBySeqNoEarlyFirstOffsetSpec
   }
 }
 
+class EventsByTagStrictBySeqNoManyInCurrentTimeBucketSpec
```

Comment:
@patriknw We've tested and used this fix for some days now. Our original issue has not occurred anymore. I would appreciate it if this PR could become a release soon. Would that be possible?